### PR TITLE
Publish nin on NPM

### PR DESCRIPTION
**Pull request by cristeahub**
nin is not published on npm. This prq adds the pieces to make that possible.

### CODE REVIEW

File: nin/backend/package.json [hunk 1]
```diff
@@ -11,10 +11,19 @@
...
     "fluent-ffmpeg": "2.0.1",
     "glob": "7.0.3",
+    "global": "4.3.1",
     "mkdirp": "0.5.1",
     "readdir": "0.0.13",
     "rimraf": "2.5.2",
     "sockjs": "0.3.16",
-    "walk": "2.3.9"
+    "walk": "2.3.9",
+    "webpack": "1.13.3"
```
Comment (stianjensen) [inline nin/backend/package.json]: Why are webpack and global needed here?

Comment (cristeahub) [inline nin/backend/package.json]: Just my setup, will update.

**Comment by cristeahub** [comment]
ping

**Comment by cristeahub** [comment]
ping

**Pull request merged**